shared/data.js

```diff
@@ -268,7 +268,7 @@ const structures = {
         'turnsToBuild': Constants.IS_PRODUCTION ? 2 : 1,
         'options': [],
         'custom': {
-            'healthMultiplier': 0.01
+            'healthMultiplier': 0.02
```

Comment:
This should be 0.01, I forgot to update the tooltip because this is a 1k building now.